item-spec/item-spec.md

```diff
@@ -210,13 +202,62 @@ Common STAC Item Media Types:
 Note: should GeoTIFF become an IANA-registered type in the future (e.g., image/geotiff), this will be added as a recommended
 media type.
 
-## Extensions
+### Item Extensions
 
 There are emerging best practices, which in time will evolve in to specification extensions for
 particular domains or uses.
 
 The [extensions page](../extensions/README.md) gives an overview about relevant extensions for STAC Items.
 
+## ItemCollection fields
```

Comment:
Wondering whether we should move that into a separate file in the item-spec folder so that ItemCollection and Item have separate files. Basically as they were before and then both are linked from in the README.md in the item-spec. Would make the individual pages more slim and helps concentrating on what people are looking for. Also, makes linking to the specs easier.

Reply:
I think it's simpler just to put them in the same file for now, since all of the other specs are one spec per directory.

Reply:
I would say one spec per file and not mixing specs. Also, the Item spec is quite long now. But just my 2 cents...

Reply:
okay, i split it out.

Reply:
Thanks. Having an item-spec.md and an itemcollection-spec.md in the same folder, borh linked from the item-spec/README:md makes moste sense to me.